src/amo/components/HomeHeroBanner/index.js

```diff
@@ -314,27 +292,16 @@ export class HomeHeroBannerBase extends React.Component<InternalProps> {
   }
 
   render() {
-    log.info('[HomeHeroBanner.render] variant is:', this.props.variant);
-
-    const homeBannerClass = makeClassName('HomeHeroBanner', {
-      'HomeHeroBanner--small': this.props.variant === AB_HOME_HERO_VARIANT_A,
-    });
-
     return (
-      <div className={homeBannerClass}>
+      <div className="HomeHeroBanner">
         <Hero name="Home" random sections={this.sections()} />
       </div>
     );
   }
 }
 
-const HomeHeroBanner: React.ComponentType<InternalProps> = compose(
-  translate(),
-  withExperiment({
-    id: AB_HOME_HERO_EXPERIMENT,
-    variantA: AB_HOME_HERO_VARIANT_A,
-    variantB: AB_HOME_HERO_VARIANT_B,
-  }),
-)(HomeHeroBannerBase);
+const HomeHeroBanner: React.ComponentType<InternalProps> = compose(translate())(
```

Comment:
You can get rid of `compose()` too and write: `... = translate()(HomeHeroBannerBase);`

Reply:
oh? So this is only used when using connect/redux?

Reply:
this is only useful when you have more than one HOC